integrations/deploy.yaml

```diff
@@ -562,14 +558,50 @@
   keywords:
     - windows
   install_description: |
-    1. Install [Windows Exporter](https://github.com/prometheus-community/windows_exporter) on every Windows host you want to monitor.
-    2. Install Netdata agent on Linux, FreeBSD or Mac.
-    3. Configure Netdata to collect data remotely from your Windows hosts by adding one job per host to windows.conf file. See the [configuration section](https://learn.netdata.cloud/docs/data-collection/monitor-anything/System%20Metrics/Windows-machines#configuration) for details.
-    4. Enable [virtual nodes](https://learn.netdata.cloud/docs/data-collection/windows-systems#virtual-nodes) configuration so the windows nodes are displayed as separate nodes.
+    Netdata offers a convenient Windows installer for easy setup. This executable provides two distinct installation modes, outlined below.
+
+    ## Graphical User Interface (GUI)
+
+    1. Download the Netdata [Windows installer](https://github.com/netdata/netdata-nightlies/releases) from the latest nightly release.
+    2. Run the `.exe` file and proceed with the installation process.
+    3. At a minimum, you will need your Netdata Cloud Space's claim token to connect your Agent to your Space.
+
+    ## Silent Mode (Command line)
+
+    If you prefer to install Netdata through the command line, you can do so by running the following command on Windows Powershell with administrator rights.
+
+    > **Note**
+    >
+    > The Windows installer is currently under beta, and thus it is only available in the nightly release channel. A stable version will be released soon.
```

Comment:
this is the snippet on the meta, as @netdata/product requested, not going in detail about the plans

Reply:
Ok, move it before `##GUI` because it is not silent-mode-specific, right?